packages/orchestrator-ui-components/src/components/WfoForms/formFields/SubscriptionField.tsx

```diff
@@ -33,12 +33,13 @@ import {
     EuiText,
 } from '@elastic/eui';
 
-import { useOrchestratorTheme } from '../../../hooks';
-import { useGetSubscriptionDropdownOptions } from '../../../hooks/surf/useGetSubscriptionDropdownOptions';
-import { SubscriptionDropdownOption } from '../../../types';
+import { useOrchestratorTheme } from '@/hooks';
+import { useGetSubscriptionDropdownOptions } from '@/hooks/surf/useGetSubscriptionDropdownOptions';
```

Comment:
Should we rename the hooks/surf folder to deprecated aswell? Also, maybe getting the subscriptionDropdown options isn't surf specific at all right?

Reply:
Let's discuss monday, sounds like something we should do.

Reply:
This seems to be the only surf specific part left, moving to deprecated folder